src/AggregateFunctions/AggregateFunctionDistinct.h

```diff
@@ -117,7 +117,7 @@ class AggregateFunctionDistinct final : public IAggregateFunctionHelper<Aggregat
         nested_func->deserialize(place, buf, arena);
     }
 
-    void insertResultInto(ConstAggregateDataPtr place, IColumn & to) const override
+    void insertResultInto(AggregateDataPtr place, IColumn & to) const override
```

Comment:
Why? All aggregate functions uses `ConstAggregateDataPtr` as it declared in `IAggregateFunction`